src/nvim/insexpand.c

```diff
@@ -640,6 +640,7 @@ static char_u *ins_compl_infercase_gettext(char_u *str, int char_len, int compl_
   while (i < char_len) {
     if (gap.ga_data != NULL) {
       ga_grow(&gap, 10);
+      assert(gap.ga_data != NULL);  // suppress clang "Dereference of NULL pointer"
```

Comment:
Regarding this one, it is maybe better to enhance the modelization we have for `ga_grow` ?
Maybe raise an error that we are OOM or something...

Reply:
Is modeling used by clang analyzer? `ga_grow()`	uses `xrealloc()`, which already errors on OOM.

Reply:
Then that must be a false positive then. I don't know how approximating is the clang analyzer.

Reply:
It is a bit weird, because `xrealloc` is actually explicitely marked as `NONNULL_RET`, so that's definitely a false positive.